### PR TITLE
Txtests2:  increases tx.py coverage to 97%

### PR DESCRIPTION
**Pull request by drmoog**
and, when a couple of questions are answered, I can probably take it to 100.

### CODE REVIEW

File: riemann/tests/tx/test_tx.py [hunk 1]
```diff
@@ -855,6 +854,59 @@ def test_sighash_forkid_all_anyone_can_pay(self):
...
+        self.outpoint = helpers.P2SH['ser']['ins'][0]['outpoint']
+
+        self.tx_in = tx.TxIn(self.outpoint, self.stack_script,
+                             self.redeem_script, self.sequence)
+        self.tx_ins = [self.tx_in]
+
+        t = tx.Tx(self.version, self.none_flag, self.tx_ins, self.tx_outs,
+                  self.none_witnesses, self.lock_time)
+
+        bob = t.script_code(index=0)
```
Comment (prestwich) [inline riemann/tests/tx/test_tx.py]: Rename variable to something clear.
Or just skip naming the variable, and include this in the asserEqual on the next line

File: riemann/tests/tx/test_tx.py [hunk 2]
```diff
@@ -855,6 +854,59 @@ def test_sighash_forkid_all_anyone_can_pay(self):
...
+        self.tx_ins = [self.tx_in]
+
+        t = tx.Tx(self.version, self.none_flag, self.tx_ins, self.tx_outs,
+                  self.none_witnesses, self.lock_time)
+
+        add_script = helpers.P2SH['ser']['ins'][0]['redeem_script']
+        adj_script_code = t._adjusted_script_code(index=0, script=add_script)
+        expected_code = tx.ByteData()
+        expected_code += tx.VarInt(len(self.redeem_script))
+        expected_code += self.redeem_script
```
Comment (prestwich) [inline riemann/tests/tx/test_tx.py]: This is basically just testing that the code does what it says it does. Instead, we should find the correct answer, create a constant in helpers.py, and assert our result is equal to the constant

File: riemann/tests/tx/test_tx.py [hunk 3]
```diff
@@ -1312,6 +1370,26 @@ def test_calculate_fee(self):
...
+        transaction = tx.DecredTx(
+            version=self.version,
+            tx_ins=[self.tx_in],
+            tx_outs=[self.tx_out],
+            lock_time=self.lock_time,
+            expiry=self.expiry,
+            tx_witnesses=[self.witness])
+
+        test_witness_hash = transaction.witness_signing_hash()
+        expected = utils.blake256(tx.DecredTx.witness_signing(transaction))
```
Comment (prestwich) [inline riemann/tests/tx/test_tx.py]: Ditto above comment. Should find a definitive answer and test that.

Comment (drmoog) [reply riemann/tests/tx/test_tx.py]: Will review

**Comment by drmoog** [comment]
Moving these issues forward

**Pull request closed without merging**